src/pages/CartPage/CartReducer.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  CartProduct,
+  ActionType,
+  ActionAdd,
+  ActionRemove,
+  ActionIncreaseAmount,
+  ActionDecreaseAmount,
+} from "./types";
+
+export const СartReducer = (
```

Comment:
It's not a component, but simple function
```suggestion
export const cartReducer = (
```